src/scripts/OSFramework/OSUI/Pattern/Tooltip/Tooltip.ts

```diff
@@ -59,10 +59,13 @@ namespace OSFramework.OSUI.Patterns.Tooltip {
 			this._tooltipBalloonPositionClass = this.configs.Position;
 		}
 
-		// Move balloon element to outside of the pattern context
+		// Move balloon element to active screen, outside of the pattern context
```

Comment:
comment:
*to the body

Reply:
The element is moved to active-screen and not to the body.